openassessment/assessment/api/peer.py

```diff
@@ -45,7 +45,7 @@ def submitter_is_finished(submission_uuid, requirements):
         bool
 
     """
-    if not requirements:
+    if requirements is None:
```

Comment:
Can the `requirements` dict never be empty? I'd prefer to keep the more general version unless there's a specific reason to change it.